app/src/components/layout/Home/HomeComponents.styled.ts

```diff
@@ -48,29 +49,28 @@ export const PageLink = styled(Link)`
 `;
 
 export const H3 = styled.h3`
-  font-weight: ${fontWeight.medium};
+  font-weight: ${fontWeight.normal};
   font-size: clamp(0.9rem, 1.35vw, 1rem);
   line-height: 1;
   padding-top: 1.45rem;
 `;
 
 export const H3Data = styled.p`
-  color: ${colors.primary};
-  font-weight: ${fontWeight.extraBold};
+  color: ${colors.black};
+  font-weight: ${fontWeight.medium};
   font-size: clamp(1.6rem, 2vw, 2.2rem);
   padding: 0.55rem 0 0.2rem 0;
 `;
 
 export const DataContext = styled.p`
-  color: ${colors.darkSupporting};
-  font-weight: ${fontWeight.medium};
+  color: ${colors.black};
   font-size: clamp(0.9rem, 1.25vw, 1rem);
 `;
 
 export const Card = styled.section`
   background: ${colors.white};
   padding: 0 2rem;
-  border: 0.063rem solid ${colors.mediumSupporting};
+  border: ${pxToRem(4)} solid #4589f6;
```

Comment:
This border seems quite large.  Have the new designs required this, I don't see any details in the ticket

Reply:
they're part of the main epic I'm working on: https://github.com/casper-network/casper-blockexplorer-frontend/issues/258